### ISSUE TITLE
Tensorflow version

### ISSUE THREAD

**Issue by himansh1314** [thatbrguy/Dehaze-GAN#12]
What's the compatible tensorflow version here? I'm using tensorflow 1.14, and have a lot of issues related to autograph and AssertionError related to batch normalization. Also there is some issue with convolution function. Please let me know the compatible tensorflow version.

**Comment by thatbrguy** [comment]
The README file mentioned that the repository uses tensorflow 1.4+. Since the project is pretty old, the tensorflow version it used it also rather old version. So I think a version closer to 1.4 should do the trick.

As of now, I do not have plans to migrate the code to a higher version of tensorflow. But feel free to try and use a higher version, or downgrade to 1.4.

**Comment by himansh1314** [comment]
The thing is, that old tensorflow versions close to 1.4 aren't available using pip. I even tried to search for .whl files but had issues.

**Comment by thatbrguy** [comment]
Ah I see. You can maybe checkout if there are any old docker images with the old tensorflow installations inside them.
